core/src/executor/evaluate/function.rs

```diff
@@ -50,6 +50,22 @@ macro_rules! eval_to_float {
 
 // --- text ---
 
+pub fn concat(name: String, exprs: Vec<Evaluated<'_>>) -> Result<Value> {
+    let mut s = String::from("");
+
+    for expr in exprs {
+        match expr.try_into()? {
+            Value::Str(value) => s.push_str(&value),
+            Value::Null => {}
```

Comment:
In MySQL, if a NULL value exists, the result string is NULL.
I think this is more natural than being treated as a blank space.

What do you think?

Reply:
While implementing the function I went with the postgres definition for concat (which ignores NULL):
https://www.postgresql.org/docs/14/functions-string.html
But you are correct for mysql the whole statement becomes NULL if a single arg is null. 
https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_concat  

I coded it this way be accident, but now I am thinking that since || returns NULL if an argument is NULL, maybe concat should do the opposite (ignores NULL) so that users have an option for either result. What do you think?

Reply:
Obviously... it would be nice to give the user an option.
How about providing it in a new PR (issue)?

Reply:
I really could care less which way we go with Concat.   If you want I can have it mimic mysql.  By the way, for most functions should we try to emulate mysql's behavior?

Reply:
no.
I think it's a process of finding the best one.

Reply:
Went ahead and changed concat so that it returns Null when concat contains a Null arg.   I will submit an issue so that we can discuss if we want this functionality or if we want to change it so that concat ignores null args